taxi_service/settings.py

```diff
@@ -38,7 +36,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "taxi",
+    "taxi.apps.TaxiConfig",
```

Comment:
You shouldn't modify the `INSTALLED_APPS`, when you declare the `taxi` app it will use `TaxiConfig` under the hood.